tailwind.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  content: [
+    './wagtail/**!(static)/*.{html,js,jsx,ts,tsx}', // Avoid purging compiled static assets
+    './client/**/*.{html,js,ts,tsx}',
+    './docs/**/*.{html,js,ts,tsx}',
+  ],
+  corePlugins: {
+    preflight: false,
+  },
+  prefix: 'w-',
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};
```

Comment:
What’s your plan to encourage usage of utility classes for logical properties, as opposed to Tailwind’s built-in `margin-` / `padding-` etc. utilities? Are we overriding the built-in utility classes with logical values, or providing additional utilities? Can we disable the built-in margin/padding?